spaghetti/network.py

```diff
@@ -36,7 +37,7 @@
     "geometries should prepare for this simply by converting "
     "to ``shapely`` geometries."
 )
-warnings.warn(f"{dep_msg}", FutureWarning)
+warnings.warn(dep_msg, FutureWarning, stacklevel=1)
```

Comment:
I think `1` this is the proper stack level here, since this is triggered on import.